lambdas/notification-configuration/index.py

```diff
@@ -0,0 +1,66 @@
+"""
+Set up indexer notification on data bucket.
+
+Remove notification on delete.
+"""
+import boto3
+
+from t4_lambda_shared.cfnresponse import send, SUCCESS, FAILED
+
+def create_mappings(params):
```

Comment:
let's call this `update_mappings` `put_mappings` or `add_mappings` so its clear that its making a remote call and not just setting a request up.